spec/requests/api/v1/events/events_spec.rb

```diff
@@ -26,3 +26,57 @@
     )
   end
 end
+
+describe 'POST/v1/events' do
+  it 'saves the address, lat, lon, name, and started_at date' do
+    event = build :event
+    event_params = {
+      address: event.address,
+      ended_at: event.ended_at,
+      lat: event.lat,
+      lon: event.lon,
+      name: event.name,
+      started_at: event.started_at,
+      owner: {device_token: event.owner.device_token
```

Comment:
Minor rubocop style ... typically have 1 space within braces:

`{ foo: bar }` 

but not [] or ()